src/Symfony/Component/Security/Core/Authorization/DebugAccessDecisionManager.php

```diff
@@ -62,7 +62,7 @@ public function decide(TokenInterface $token, array $attributes, $object = null)
      */
     public function setVoters(array $voters)
     {
-        if (!$this->manager instanceof AccessDecisionManager) {
+        if (!method_exists($this->voters, 'setVoters')) {
```

Comment:
Am I missing something or you are calling `method_exists` on an array?
Should it not be `!method_exists($this->manager, 'setVoters')` instead?